packages/spinners/README.md

```diff
@@ -0,0 +1,8 @@
+# `@truffle/spinners`
```

Comment:
Is there a way to make this package more general purpose than just spinners? Maybe @truffle/console-utils?

Reply:
I considered that, but decided against it for two reasons:

1. Keeping it in a dedicated module keeps the dependency tree lighter for modules that only want this functionality (though hopefully this can be improved via tree shaking when we switch over to ESMs)
2. I didn't want to create a module with a fuzzy boundary around its functional scope that might grow organically in such a way that would thwart a more considered presentation-layer design/architecture effort.